db/migrate/20140108010302_add_token_to_projects.rb

```diff
@@ -2,7 +2,7 @@ class AddTokenToProjects < ActiveRecord::Migration
   def change
     change_table :projects do |t|
       t.string :token
-      t.index :token
+      t.index :token, length: 191
```

Comment:
indices are limited to 767 bytes in mysql and each character is 4 bytes, 767/4 = 191.75. Unfortunately Rails just defaults to 255.